errors/Readme.md

```diff
@@ -6,6 +6,10 @@
   - Spec: In Progress 🚧
   - Implementation: Future 👽
 
+Definition of errors in Prisma Framework. (In this document we make the distinction between [Unknown Errors](https://github.com/prisma/specs/tree/master/errors#unknown-errors) and [Known Errors](https://github.com/prisma/specs/tree/master/errors#known-errors).)
```

Comment:
These links will probably break after merge.
I think you can get totally rid of that second sentence, better add more to the first one where these errors (between which components) are relevant and if those are shown to the user.